### DISCUSSION TITLE
day 23 1번문제

### DISCUSSION THREAD

**Discussion by Mang-Joo** [I-JUNNYLAND-I/algorithm#131] [Q&A] [answered]
문제 이해를 못하겠습니다 ㅠㅠ

**Comment by Junnyjun** [comment, marked as answer]
Low -> 5 
High -> 10
인경우

5 + 6 + 7 + 8 + 9 + 10 이렇게
노드를 순회해서 범위에 속하는 모든값들을 더하면 됩니다

**Reply by Mang-Joo** [reply]
감사합니다 오후에 한번 풀어보겠습니다.
